### PR TITLE
Store window ref in embed manager

### PR DESCRIPTION
**Pull request by EnoF**

### CODE REVIEW

File: sdk/src/embed-manager.ts [hunk 1]
```diff
@@ -63,8 +65,13 @@ export class EmbedManager {
     // for issue that the popup may be open somewhere in an (invisible to the user) tab, first try to close the popup.
     // inivisble because it could be below another application or minimized on desktop
     // or in a other tab on iOS
-    this.closePopup();
-    this.popup = open(this.baseUrl + path, 'SpireKeyPopup', params);
+    setTimeout(() => {
```
Comment (Randynamic) [inline sdk/src/embed-manager.ts:68]: ❤️

**Review by Randynamic — APPROVED** [review]

**Comment by Randynamic** [comment]
works like a charm again

File: sdk/src/embed-manager.ts (hunk 1 above)
Comment (sstraatemans) [reply sdk/src/embed-manager.ts:69]: 🤯

Comment (EnoF) [reply sdk/src/embed-manager.ts:69]: :sob:

**Pull request merged**